scripts/travis/start_elasticdl_job.sh

```diff
@@ -0,0 +1,56 @@
+JOB_TYPE=$1
+
+kubectl apply -f elasticdl/manifests/elasticdl-rbac.yaml
```

Comment:
I'd thought that this line should be right after the starting of the minikube cluster.  We don't need to run it multiple times.

Reply:
Yeah!